Backend/WolfDen.Domain/Entity/Employee.cs

```diff
@@ -0,0 +1,54 @@
+﻿using EmployeeManagementDomain.Entity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WolfDen.Domain.Entity
+{
+    public class Employee
+    {
+        public int Id { get; private set; }
+        public int EmployeeId { get; private set; }
```

Comment:
Id and EmployeeId?

Reply:
id table id primary key and EmployeeId is Unique id

Reply:
Use Identifier instead of Id